plugins/woocommerce/includes/admin/meta-boxes/class-wc-meta-box-order-data.php

```diff
@@ -250,6 +250,24 @@ public static function output( $post ) {
 
 					?>
 				</p>
+				<?php
+					/**
+					 * Hook allowing extenders to render custom content
+					 * within the Order details box.
+					 *
+					 * This allows urgent notices or other important
+					 * order-related info to be displayed upfront in
+					 * the order page. Example: display a notice if
+					 * the order is disputed.
+					 *
+					 * WC_Admin_Notices::render_contextual_notice() can be
+					 * used to render a prominent notice with a CTA button.
```

Comment:
@haszari This method got removed, right?

Reply:
Thanks for catching this ! I'll remove and re-check to ensure there aren't any other remnants.

Reply:
Removed! e8f2d3f0618040dbe421177fe6f5d09d59fe39a9 

Should be good to merge @coreymckrill  – let me know if there's anything else.